operations/_scripts/generate/generate_tf_vars.sh

```diff
@@ -62,207 +73,81 @@ else
-if [[ -n "$AWS_SECRET_ENV" ]]; then
-  aws_secret_env="aws_secret_env = \"${AWS_SECRET_ENV}\""
-fi
-
-aws_ami_id=
-if [[ -n "$AWS_AMI_ID" ]]; then
-  aws_ami_id="aws_ami_id = \"${AWS_AMI_ID}\""
-fi
-
-lb_port=
-if [[ -n "$LB_PORT" ]]; then
-  lb_port="lb_port = \"$LB_PORT\""
-fi
-
-lb_healthcheck=
-if [[ -n "$LB_HEALTHCHECK" ]]; then
-  lb_healthcheck="lb_healthcheck = \"$LB_HEALTHCHECK\""
-fi
-
-lb_access_bucket_name=
-if [[ -n "$LB_LOGS_BUCKET" ]]; then
-  lb_access_bucket_name="lb_access_bucket_name = \"${LB_LOGS_BUCKET}\""
-fi
-
-# Should this be moved to a defaults?
-ops_repo_environment="ops_repo_environment = \"deployment\""
-security_group_name="security_group_name = \"${GITHUB_IDENTIFIER}\""
-app_install_root="app_install_root = \"/home/ubuntu\""
-
-app_org_name=
-if [[ -n "$GITHUB_ORG_NAME" ]]; then
-  app_org_name="app_org_name = \"${GITHUB_ORG_NAME}\""
-fi
-
-app_repo_name=
-if [[ -n "$GITHUB_REPO_NAME" ]]; then
-  app_repo_name="app_repo_name = \"${GITHUB_REPO_NAME}\""
-fi
-
-app_branch_name=
-if [[ -n "$GITHUB_BRANCH_NAME" ]]; then
-  app_branch_name="app_branch_name = \"${GITHUB_BRANCH_NAME}\""
+sub_domain_name=
+if [ -n "$SUB_DOMAIN" ]; then
+  sub_domain_name="sub_domain_name = \"$SUB_DOMAIN\""
+else
+  sub_domain_name="sub_domain_name = \"$GITHUB_IDENTIFIER\""
 fi
 
-create_keypair_sm_entry=
-if [[ -n "$CREATE_KEYPAIR_SM_ENTRY" ]]; then
-  create_keypair_sm_entry="create_keypair_sm_entry = \"${CREATE_KEYPAIR_SM_ENTRY}\""
+aws_postgres_subnets=
+if [ -n "${AWS_POSTGRES_SUBNETS}" ]; then
+  aws_postgres_subnets="aws_postgres_subnets = \"$(comma_str_to_tf_array $AWS_POSTGRES_SUBNETS)\""
 fi
+echo "AWS Postgres subnets: $aws_postgres_subnets"
 
-# RDS
-
-postgres_subnets=
-if [ -n "${POSTGRES_SUBNETS}" ]; then
-  postgres_subnets="postgres_subnets = \"$(comma_str_to_tf_array $POSTGRES_SUBNETS)\""
-fi
 
-
-echo "Postgres subnets: $postgres_subnets"
-
-#security_group_name_pg=
-#if [ -n "${POSTGRES_SUBNETS}" ]; then
-  security_group_name_pg="security_group_name_pg = \"${GITHUB_IDENTIFIER}-pg\""
-#fi
-enable_postgres=
-if [ -n "${ENABLE_POSTGRES}" ]; then
-  enable_postgres="enable_postgres = \"${ENABLE_POSTGRES}\""
-fi
-postgres_engine=
-if [ -n "${POSTGRES_ENGINE}" ]; then
-  postgres_engine="postgres_engine = \"${POSTGRES_ENGINE}\""
-fi
-postgres_engine_version=
-if [ -n "${POSTGRES_ENGINE_VERSION}" ]; then
-  postgres_engine_version="postgres_engine_version = \"${POSTGRES_ENGINE_VERSION}\""
-fi
-postgres_instance_class=
-if [ -n "${POSTGRES_INSTANCE_CLASS}" ]; then
-  postgres_instance_class="postgres_instance_class = \"${POSTGRES_INSTANCE_CLASS}\""
-fi
-postgres_database_name=
-if [ -n "${POSTGRES_DATABASE_NAME}" ]; then
-  postgres_database_name="postgres_database_name = \"${POSTGRES_DATABASE_NAME}\""
-fi
-postgres_database_port=
-if [ -n "${POSTGRES_DATABASE_PORT}" ]; then
-  postgres_database_port="postgres_database_port = \"${POSTGRES_DATABASE_PORT}\""
+#-- Application --#
+app_port=$(generate_var app_port $APP_PORT)
```

Comment:
Will these be null if no value is provided?

Reply:
Exactly.